library/plugins/influxdb/influxdb2-publisher/src/main/kotlin/io/github/cdsap/talaiot/publisher/influxdb2/InfluxDb2PublisherConfiguration.kt

```diff
@@ -0,0 +1,67 @@
+package io.github.cdsap.talaiot.publisher.influxdb2
+
+import io.github.cdsap.talaiot.configuration.PublisherConfiguration
+import io.github.cdsap.talaiot.metrics.BuildMetrics
+import io.github.cdsap.talaiot.metrics.TaskMetrics
+
+/**
+ * Configuration for the [InfluxDb2Publisher]. It belongs to the Publisher configurations
+ *
+ * influxDbPublisher {
+ *    token = "tracking"
+ *    org = "url"
+ *    bucket = "tracking"
+ *    buildMetricName = "buildMetric"
+ *
+ * }
+ */
+class InfluxDb2PublisherConfiguration : PublisherConfiguration {
+
+    override var publishBuildMetrics: Boolean = true
+    override var publishTaskMetrics: Boolean = true
+
+    /**
+     * name of the publisher
+     */
+    override var name: String = "influxDb2"
+
+    /**
+     * influx instance auth token
+     */
+    var token: String = ""
+
+    /**
+     * influx instance bucket name
+     */
+    var bucket: String = ""
+
+    /**
+     * influx instance org name
+     */
+    var org: String = ""
```

Comment:
do you think the properties `bucket`or `org` can have default values provided by the lib? Or do you think is better to make them mandatory in the dsl conf?

Reply:
It should be a mandatory fields